### PR TITLE
bugfix #1308: typo in flash message after incorrect login.

### LINKED ISSUE #1308: Typo in flash message after incorrect login.
### Steps to reproduce the error

- Login with wrong credentials.

### Actual behavior

screenshot from 2017-03-16 10-57-17

### Expected behavior

screenshot from 2017-03-16 11-24-11

### PR DESCRIPTION
**Pull request by rishabhiitbhu**
### Proposed changes in this pull request

Addressed issue #1308 in this PR.

-  Created new class `LoginForm` in `accounts/forms.py` inheriting `LoginForm` of `allauth/account/forms.py` [ link to allauth/account/forms.py [here](https://github.com/pennersr/django-allauth/blob/master/allauth/account/forms.py#L69) ]
- Edited `ACCOUNT_FORMS` dictionary of `config/settings/default.py` by adding new key `login` with `accounts.forms.LoginForm` as value. [as according to allauth docs [here](https://django-allauth.readthedocs.io/en/latest/configuration.html)]
- Added new `LoginFormTest`  class with `test_login_error_message` method in `accounts/tests/test_forms.py` to assert the changes.

### When should this PR be merged

- After It passes the tests.
- After Review from mentors.

### Risks

- None

### Follow up actions

- None

#### General

- [ ] **Is this PR explained thoroughly?** All code changes must be accounted for in the PR description. 
- [ ] **Is the PR labeled correctly?** It should have the `migration` label if a new migration is added. 
- [ ] **Is the risk level assessment sufficient?** The risks section should contain all risks that might be introduced with the PR and which actions we need to take to mitigate these risks. Possible risks are database migrations, new libraries that need to be installed or changes to deployment scripts.

#### Functionality

- [ ] **Are all requirements met?** Compare implemented functionality with the requirements specification.
- [ ] **Does the UI work as expected?** There should be no Javascript errors in the console; all resources should load. There should be no unexpected errors. Deliberately try to break the feature to find out if there are corner cases that are not handled. 

#### Code

- [ ] **Do you fully understand the introduced changes to the code?** If not ask for clarification, it might uncover ways to solve a problem in a more elegant and efficient way.
- [ ] **Does the PR introduce any inefficient database requests?** Use the debug server to check for duplicate requests. 
- [ ] **Are all necessary strings marked for translation?** All strings that are exposed to users via the UI must be [marked for translation](https://docs.djangoproject.com/en/1.10/topics/i18n/translation/). 

#### Tests

- [ ] **Are there sufficient test cases?** Ensure that all components are tested individually; models, forms, and serializers should be tested in isolation even if a test for a view covers these components. 
- [ ] **If this is a bug fix, are tests for the issue in place**  There must be a test case for the bug to ensure the issue won’t regress. Make sure that the tests break without the new code to fix the issue.
- [ ] **If this is a new feature or a significant change to an existing feature** has the manual testing spreadsheet been updated with instructions for manual testing?

#### Documentation

- [ ] **Are changes to the UI documented in the platform docs?** If this PR introduces new platform site functionality or changes existing ones, the changes must be documented in the [Cadasta Platform Documentation](https://github.com/Cadasta/cadasta-docs).
- [ ] **Are changes to the API documented in the API docs?** If this PR introduces new API functionality or changes existing ones, the changes must be documented in the [API docs](https://github.com/Cadasta/api-docs).
- [ ] **Are reusable components documented?** If this PR introduces components that are relevant to other developers (for instance a mixin for a view or a generic form) they should be documented in the Wiki.

### CODE REVIEW

File: cadasta/accounts/forms.py [hunk 1]
```diff
@@ -108,6 +108,22 @@ def save(self, *args, **kwargs):
         return user
 
 
+class LoginForm(allauth_forms.LoginForm, forms.Form):
+    error_messages = {
+        'account_inactive':
```
Comment (wonderchook) [inline cadasta/accounts/forms.py]: We would never tell someone the account was inactive in this way. So I think this is unnecessary. 

Comment (rishabhiitbhu) [inline cadasta/accounts/forms.py]: @wonderchook thanks for your review. Should I remove only `account_inactive` message or all error_messages except the `username_email_password_mismatch` ? As we'll be needing this message only as we have `ACCOUNT_AUTHENTICATION_METHOD` set to `"username_email` in `config/settings/default.py` [link](https://github.com/Cadasta/cadasta-platform/blob/master/cadasta/config/settings/default.py#L136)

**Comment by rishabhiitbhu** [comment]
@wonderchook Should I remove the `LoginFormTest` from `cadasta/accounts/tests/test_forms.py` also ?

**Comment by wonderchook** [comment]
Yes, just change the text. Sorry I didn't pick this up earlier, since the bug was just saying there was a typo I didn't read as closely as I should have.

**Comment by rishabhiitbhu** [comment]
@wonderchook Okay. I will remove the `LoginFormTest`  but we'll have to keep the `ACCOUNT_FORMS`  dictionary of `config/settings/default.py` with `"login"` key pointing to `'accounts.forms.LoginForm'` as according to allauth doc [here](https://django-allauth.readthedocs.io/en/latest/configuration.html). I'll resubmit the PR.

**Comment by seav** [comment]
@rishabhiitbhu, thank you for your contribution. Unfortunately, I don't think your proposed solution is correct. Please see my comment on #1308 for more details.

**Pull request closed without merging**